tools/IceRpc.Slice.Tools/SliceCCSharpTask.cs

```diff
@@ -26,6 +26,12 @@ public class SliceCCSharpTask : ToolTask
     /// to <c>-R</c> slicec-cs compiler option.</summary>
     public string[] References { get; set; } = Array.Empty<string>();
 
+    /// <summary>
+    /// The RPC provider to generate code for, corresponds to the <c>--rpc</c> slicec-cs compiler option.
+    /// </summary>
+    [Required]
+    public string Rpc { get; set; } = "";
```

Comment:
The `[Required]` attribute does not suppress the need to have a default value for this property?

Reply:
No, this only cause an error if the task caller doesn't provide the parameter.

Without the value I get:

```
warning CS8618: Non-nullable property 'Rpc' must contain a non-null value when exiting constructor
```